src/time_integrator.f90

```diff
@@ -5,18 +5,23 @@ module m_time_integrator
 
   implicit none
 
-  private adams_bashforth
+  private adams_bashforth, runge_kutta
 
   type :: time_intg_t
-    integer :: istep, order, nvars, nolds
+    integer :: method, istep, istage, order, nstep, nstage, nvars, nolds
     real(dp) :: coeffs(4, 4)
+    real(dp) :: rk_b(4, 4)
+    real(dp) :: rk_a(3, 3, 4)
+    character(len=3) :: sname
```

Comment:
Add a member variable to store the method name, useful to keep track of the time-integration approach used, specially for time-convergence study